src/main/java/com/loadero/model/FileParams.java

```diff
@@ -172,6 +174,10 @@ public FileParamsBuilder withPassword(String password) {
          * @return Built {@link FileParams} object.
          */
         public FileParams build() {
+            if (StringUtil.empty(this.name)) {
```

Comment:
for other types i see this only being done in the `withName` functions. either there is something i'm missing or this should be done globally (and then in a separate task)